### PR TITLE
OS-40: Theming and Customization

### PR DESCRIPTION
**Pull request by shayaanmiyy**
# Description

Customizing the Storybook UI

- Theming Storybook to match WordPress themes

- Branding Storybook for project-specific needs

### Acceptance Criteria:

- Customization of the Storybook UI to match project themes

- Implementation of branding in Storybook

### Exercise:

- Customize the Storybook UI to match a WordPress theme's color scheme.

- Add project-specific branding to the Storybook interface

image

I have Customize the Storybook to match with the DP Color Palate

### CODE REVIEW

File: web/app/themes/dp-timber-theme/.storybook/manager.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    //
+    colorPrimary: "#3A10E5",
+    colorSecondary: "#0a6aff",
+
+    // UI
+    appBg: "#ffffff",
+    appContentBg: "#ffffff",
+    appPreviewBg: "#ffffff",
+    appBorderColor: "#585C6D",
+    // appBorderColor: "#0a6aff",
```
Comment (Prashant114606) [inline web/app/themes/dp-timber-theme/.storybook/manager.js]: can you remove this commented code?

Comment (shayaanmiyy) [reply web/app/themes/dp-timber-theme/.storybook/manager.js]: Yes I will Remove

File: web/app/themes/dp-timber-theme/.storybook/manager.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  theme: create({
+    base: "light",
+
+    brandTitle: "My custom Storybook",
+    brandUrl: "https://digitalpolygon.com",
+    brandImage: "https://www.digitalpolygon.com/DigitalPolygon-logo.svg",
+    brandTarget: "_self",
+
+    //
+    colorPrimary: "#3A10E5",
```
Comment (Prashant114606) [inline web/app/themes/dp-timber-theme/.storybook/manager.js:14]: can you also explain what all these variable do ?

Comment (shayaanmiyy) [reply web/app/themes/dp-timber-theme/.storybook/manager.js:14]: These variable defined by the storybook to change the UI color of the Storybook, If variable name changes the changes will not work

**Pull request merged**